Documentation/tutorials/kubernetes/configure-kubectl.md

```diff
@@ -42,5 +42,5 @@ Any connection errors will indicate that your cluster is not yet done bootstrapp
 For more information on troubleshooting cluster errors, see [triaging a cluster][triaging-cluster].
 
 
-[triaging-cluster]: ../../troubleshooting/troubleshooting.html
```

Comment:
rather, these should have file extension `.md`, rather than `.html`.

(General rule: prefer relative links, then links to full github urls of external repos' docs, then absolute references into the coreos.com name space, roughly in that order. Target the .md source except in the final case. 
Result: link works when reading on github, and `pages/sync` rewrites the first and second cases to work on coreos.com.)